jbake-core/build.gradle

```diff
@@ -1,32 +1,38 @@
 import java.time.format.DateTimeFormatter
 
-apply from: "$rootDir/gradle/sonarqube.gradle"
-apply plugin: 'java-library'
+plugins {
+    id "org.jbake.convention.java-common"
+    id 'java-library'
+    id 'org.ajoberstar.grgit' version "$grgitVersion"
+    id 'nebula.optional-base' version '3.0.3'
+}
 
+apply from: "$rootDir/gradle/maven-publishing.gradle"
+apply from: "$rootDir/gradle/signing.gradle"
 
 description = "The core library of JBake"
 
 dependencies {
-    implementation "commons-io:commons-io:$commonsIoVersion"
-    implementation "org.apache.commons:commons-configuration2:$commonsConfigurationVersion"
+    api "commons-io:commons-io:$commonsIoVersion"
+    api "org.apache.commons:commons-configuration2:$commonsConfigurationVersion"
     implementation "commons-beanutils:commons-beanutils:$commonsBeanutilsVersion"
-    implementation "org.apache.commons:commons-vfs2:$commonsVfs2Version", optional
+    implementation "org.apache.commons:commons-vfs2:$commonsVfs2Version"
```

Comment:
Was this optional removed intentionally?

Reply:
No, let me restore that

Reply:
Fixed.

Reply:
:+1: